spec/integration/search/aggregates_spec.rb

```diff
@@ -0,0 +1,252 @@
+require 'spec_helper'
+
+RSpec.describe 'search queries' do
+  context 'with aggregates' do
+    it "returns document count" do
+      populate_content_indexes(section_count: 2)
```

Comment:
I'm not sure I'm following what the test is doing, without knowing what `populate_content_indexes` does. Why do we expect 2 documents in each browse page? Would this number change if we removed the number of indexes?

I think we could probably make the setup clearer by not using this method (for example, by making a method call for each document instead).

Reply:
Have renamed the method to be clearer. let me know if you still prefer inlining.